clients/lcdproc/lcdproc.conf

```diff
@@ -106,7 +106,8 @@ DateFormat="%x"
 # Display the title bar in two-line mode. Note that with four lines or more
 # the title is always shown. [default: true; legal: true, false]
 #ShowTitle=false
-
+# Enable or disable the backlight
```

Comment:
Could you add `[default: false; legal: true, false]` in this comment ?

This helps [cme](https://github.com/dod38fr/config-model/wiki/Managing-Lcdproc-configuration-with-cme) to [generate a more accurate model for lcdproc](https://ddumont.wordpress.com/2011/07/03/generate-a-configuration-editor-from-a-config-template-file-with-perl-lcdproc-example/)

All the best

Reply:
This has been added in the second commit.